docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1262 @@
+  buckets elsewhere as well, ensure that this bucket name is unique.
+
+### Telemetry configuration
+
+Telemetry is the process of collecting and analyzing application data to
+gain insights into system performance, user behavior, resource usage, and more.
+
+Enduro includes a GoLang [OpenTelemetry] package, which when enabled and
+properly configured, can support the generation, collection, and export of
+telemetry data from Enduro for debugging, optmization, and development purposes.
+At this time, no user data is reviewed or collected by the Enduro telemetry
+package.
+
+!!! tip
+
+    For additional information on logging, see:
+
+    * [Logging](../dev-manual/logging.md)
+
+**Default values**:
+
+```toml
+[telemetry.traces]
+enabled = false
+address = ""
+samplingRatio = 1.0
+```
+
+* `enabled`: Boolean value that enables or disables telemetry in Enduro. Set to
+  `false` by default - change to `true` to enable.
+* `address`: the gRPC address and port of the related observability tool used
+  for parsing the collected telemetry data. There are many different tools (both
+  proprietary and open source) that can be used with OpenTelemetry data - see
+  the [OpenTelemetry docs] for a list of vendor tools known to be compatible.
+* `samplingRatio`: This ratio defines the frequency and granularity of data
+  collection, with 1.0 representing a 100% capture of all events regardless of
+  repetition. In many cases an internal event or call may be repeated dozens or
+  even hundreds of times, leading to redundancy in the captured logs. A
+  developer or system administrator may then wish to reduce the logging of
+  similar events by lowering the sampling ratio - for example, if the ratio is
+  reduced to 0.5 (i.e. 50% sampling ratio), then a single event that occurs 10
+  times would only be logged 5 times in the resulting trace data.
+
+### Preprocessing child workflow configuration
+
+The following two sections are for configuring a [child workflow] as part of the
+ingest workflow run by Enduro. A child workflow is a concept borrowed from
+Enduro's [workflow engine], [Temporal] where one workflow spawns an ancillary
+workflow. We use this model in Enduro to keep custom activities with
+organization-specific business rules or policy requirements separated from the
+underlying general Enduro code. For more information, see:
+
+* [Preprocessing child workflow](../dev-manual/preprocessing.md)
+
+**Default values**:
+
+```toml
+[preprocessing]
+enabled = false
+extract = false
+sharedPath = "/home/enduro/preprocessing"
+```
+
+* `enabled`: Boolean value, set to `false` by default. To enable and configure a
+  custom child workflow, change to `true`.
+* `extract`: Boolean value, set to `false` by default. This setting determines
+  whether SIP extraction happens as part of the child workflow or not. When set
+  to false, SIP extraction will occur at the beginning of the workflow. In some
```

Comment:
"When set to false, SIP extraction will occur in the parent Enduro workflow before the child workflow is run."

Reply:
This hasn't been changed.

Reply:
Fixed now!